src/lib/Menu/URLWildcardEditRightSidebarBuilder.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Menu;
+
+use EzSystems\EzPlatformAdminUi\Menu\Event\ConfigureMenuEvent;
+use EzSystems\EzPlatformAdminUi\Tab\URLManagement\URLWildcardsTab;
+use JMS\TranslationBundle\Model\Message;
+use JMS\TranslationBundle\Translation\TranslationContainerInterface;
+use Knp\Menu\ItemInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class URLWildcardEditRightSidebarBuilder extends AbstractBuilder implements TranslationContainerInterface
```

Comment:
```suggestion
final class URLWildcardEditRightSidebarBuilder extends AbstractBuilder implements TranslationContainerInterface
```

Recommended method of customizing menus is using event listeners so class could be marked as final.